assets/scss/_explorer.scss

```diff
@@ -120,7 +120,9 @@
     }
 
     .selection {
-      p.help-text {
+      .helptext {
```

Comment:
If you use the foundation template, you won't need this.

Reply:
indeed